services/api/sample.env

```
@@ -60,6 +60,10 @@ CSM_AUTH_ENABLED=true
 LOGS_LINK= #URL to logs for api, included in error email. Example: https://console.cloud.google.com/run/detail/us-central1/rsu-manager-cloud-run-api/logs?authuser=1&project=cdot-oim-cv-dev
 ENVIRONMENT_NAME= #Environment name, just to display in email. Example: cdot-oim-cv-dev
 
+# Error Email Contact Configuration
+LOGS_LINK= #URL to logs for api, included in error email. Example: https://console.cloud.google.com/run/detail/us-central1/rsu-manager-cloud-run-api/logs?authuser=1&project=cdot-oim-cv-dev
+ENVIRONMENT_NAME= #Environment name, just to display in email. Example: cdot-oim-cv-dev
+
```

chore(blocking): it looks like these env vars are already defined. let's remove the duplicates
```suggestion
LOGS_LINK= #URL to logs for api, included in error email. Example: https://console.cloud.google.com/run/detail/us-central1/rsu-manager-cloud-run-api/logs?authuser=1&project=cdot-oim-cv-dev
ENVIRONMENT_NAME= #Environment name, just to display in email. Example: cdot-oim-cv-dev
```